client/state/action-types.js

```diff
@@ -480,6 +480,10 @@ export const MEMBERSHIPS_CONNECTED_ACCOUNTS_RECEIVE = 'MEMBERSHIPS_CONNECTED_ACC
 export const MEMBERSHIPS_PRODUCTS_RECEIVE = 'MEMBERSHIPS_PRODUCTS_RECEIVE';
 export const MEMBERSHIPS_PRODUCTS_LIST = 'MEMBERSHIPS_PRODUCTS_LIST';
 export const MEMBERSHIPS_PRODUCT_RECEIVE = 'MEMBERSHIPS_PRODUCT_RECEIVE';
+export const MEMBERSHIPS_SUBSCRIPTIONS_LIST = 'MEMBERSHIPS_SUBSCRIPTIONS_LIST';
+export const MEMBERSHIPS_SUBSCRIPTIONS_RECEIVE = 'MEMBERSHIPS_SUBSCRIPTIONS_RECEIVE';
+export const MEMBERSHIPS_SUBSCRIPTIONS_LIST_SUCCESS = 'MEMBERSHIPS_SUBSCRIPTIONS_LIST_SUCCESS';
+export const MEMBERSHIPS_SUBSCRIPTIONS_LIST_FAILURE = 'MEMBERSHIPS_SUBSCRIPTIONS_LIST_FAILURE';
```

Comment:
we've been trying to eliminate actions ending in `_RECEIVE` and `_SUCCESS` and `_FAILURE`

can you update this to use the data layer? I'd be happy to help guide you. should end up with less code.

Reply:
Done as you propose in https://github.com/Automattic/wp-calypso/pull/26023